Multisig/UI/App/MainTabBarViewController.swift

```diff
@@ -77,6 +78,8 @@ class MainTabBarViewController: UITabBarController {
         appearsFirstTime = false
 
         onFirstAppear(self)
+
+        WhatsNewHandler().whatsNewViewController?.present(on: self)
```

Comment:
Is this takes into consideration the priority for Enter passcode screen, update available screen and terms accept screen ?

Reply:
Terms accept & passcode come before this screen. 
Update available comes after this screen IIRC.